apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -94,6 +97,15 @@ export const makeHomepageProps = (
       })
     ),
   },
+  webinars: strapiHomepage.data.attributes.webinars.data.map((webinar) => ({
+    ...webinar.attributes,
+    isVisibleInHome: true,
```

Comment:
I think this attribute should be removed from here and from `Webinar` type. I think is better to add  `isVisibleInHome` only in [`apps/nextjs-website/src/_contents/webinars.ts` ](https://github.com/pagopa/developer-portal/blob/9c447fd5406b394804f7883f5d2f02564bca31ce/apps/nextjs-website/src/_contents/webinars.ts#L41C36-L41C43) constant to manange static webinars like you did in line 118.
Something like this:
```
type StaticWebinar = Webinar & {isVisibleInHome: boolean}
export const webinars: readonly StaticWebinar[] = [
...
```